### DISCUSSION TITLE
Invalid docs for gitTimeout

### DISCUSSION THREAD

**Discussion by wwuck** [renovatebot/renovate#15169] [Request Help] [answered]
I was reading the [docs for gitTimeout](https://docs.renovatebot.com/self-hosted-configuration/#gittimeout) and there is conflicting information.  The default value is listed as `10000` but the description says `The value must be between 2000 and 6000 (milliseconds) inclusive.`

What is the correct information?

**Comment by hasanwhitesource** [comment, marked as answer]
The upper bound is a typo. it should be `60000`.

**Reply by HonkingGoose** [reply]
Docs are fixed with PR #15171 now.
